app/services/rapid_ready_for_decision/rrd_special_issue_manager.rb

```diff
@@ -21,11 +21,13 @@ def add_special_issue
 
     private
 
-    HYPERTENSION_DISABILITY = RapidReadyForDecision::Constants::DISABILITIES[:hypertension]
+    RRD_DIAGNOSTIC_CODES = RapidReadyForDecision::Constants::DISABILITIES_BY_CODE.keys
```

Comment:
Interesting that the linter didn't ask for a `.freeze` for this constant.

Reply:
Will go ahead and add that!

Reply:
Yeahhh interesting. In my case it gave a syntax error for 
```ruby
ASTHMA_KEYWORDS = RapidReadyForDecision::Constants::DISABILITIES[:asthma][:keywords] # syntax error: dynamic constant assignment
```

Maybe it's considered "dynamic" if it's nested keys (`DISABILITIES[:asthma][:keywords]`)